SasquatchMac/SasquatchMac/TransmissionTarget.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+
+private let kDefaultTargetKey = "defaultTargetKey"
+
+class TransmissionTargets {
+  static let shared = TransmissionTargets.init()
+
+  var transmissionTargets = [String: MSAnalyticsTransmissionTarget]()
+  let defaultTransmissionTargetIsEnabled: Bool
+  private var sendsAnalyticsEvents = [String: Bool]()
+
+  private init() {
+
+    // Default target.
+    let startTarget = UserDefaults.standard.integer(forKey: kMSStartTargetKey)
+    defaultTransmissionTargetIsEnabled = startTarget == 1 || startTarget == 2
```

Comment:
Why are we using 1 and 2 instead of a constant or something else to identify the start targets?

Reply:
Added the enum `StartupMode` for it.